luigi_pipeline/lib/model/base_mt_schema.py

```diff
@@ -28,6 +28,18 @@ def __repr__(self):
             requires = f' (requires: {", ".join(self.requirements)})'
         return f"{self.name}{requires}"
 
+    def __call__(self: "RowAnnotation", schema: "BaseMTSchema", overwrite: bool = False):
```

Comment:
Several of the tests expected these to be callable... at this point I'd vote to just remove the tests but that also kind of seems like a bad idea 🤷.

Reply:
Those tests look ugly. It may be better if the tests are done at a higher level (e.g., calling `annotate_all()` rather than each annotation function). But I like your change that makes the annotation callable.

Reply:
Yep!  Agreed.  My assumption when starting this was that we should try to keep the tests and manipulate the code so that the tests passed... but I think the better answer here is probably to just get rid of the tests that called annotations individually (which I think was most of them :() .